qemu/tests/cfg/balloon_service.cfg

```diff
@@ -32,10 +31,16 @@
     balloon_type_enlarge = enlarge
     variants:
         - small_polling_interval:
+            type = balloon_service
             polling_interval = 2
         - large_polling_interval:
+            type = balloon_service
             polling_interval = 21474836
```

Comment:
I think you can leave "type = balloon_service" as before, only change type in variants "sc_interrogate" is enough.